pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -108,7 +112,7 @@ type ActualStateOfWorld interface {
 	// volumes, depend on this to update the contents of the volume.
 	// All volume mounting calls should be idempotent so a second mount call for
 	// volumes that do not need to update contents should not fail.
-	PodExistsInVolume(podName volumetypes.UniquePodName, volumeName v1.UniqueVolumeName) (bool, string, error)
+	PodExistsInVolume(vmt VolumeToMount) (bool, string, error)
```

Comment:
Don't collapse - scope the input for each method down to the bare minimum it needs.

Reply:
In general I agree but if we keep each argument separate then this function will now need 4 arguments and IMO i would be happy if it were 3 but 4 is tad too much. 

I know in other places we have used https://refactoring.guru/introduce-parameter-object pattern when number of parameters to a function grow too big, for example `MarkVolumeAsMounted` will require 9 arguments if specified separately.